etl/steps/data/garden/met_office_hadley_centre/2023-01-02/near_surface_temperature.meta.yml

```diff
@@ -0,0 +1,34 @@
+dataset:
+  title: Near-surface temperature anomaly (Met Office Hadley Centre, 2022)
+  description: |
+    Temperature anomalies are based on the HadCRUT5 near-surface temperature dataset as published by the Met Office Hadley Centre. Temperature anomalies are given in degrees celsius relative to the average temperature over the period 1961-1990.
+
+    These are available for the Northern Hemisphere and the Southern Hemisphere. The global mean has been calculated by averaging anomalies for northern and southern hemispheres.
+  sources:
+    -
+      name: Met Office Hadley Centre
```

Comment:
Maybe worth adding in the year here so it's Met Office Hadley Centre (2022). Perhaps even better to use the citation given [here?](https://www.metoffice.gov.uk/hadobs/hadcrut5/terms_and_conditions.html)

Reply:
Ah I see you have the Morice one below :)